messages/src/message.rs

```diff
@@ -108,6 +108,24 @@ pub enum HostRequest {
     /// Request that the modules be reset.
     Reset,
 
+    /// Request that the ResetL line be asserted.
+    AssertReset,
+
+    /// Request that the ResetL line be de-asserted.
+    DeassertReset,
+
+    /// Request that the LpMode line be asserted.
+    AssertLpMode,
+
+    /// Request that the LpMode line be de-asserted.
+    DeassertLpMode,
+
+    /// Request that power be enabled to a module should it be inserted.
```

Comment:
What happens if there is no module inserted? Does this return an error message?

Reply:
It does not. The `should it be inserted` is loadbearing here as this message does not explicitly turn on/off the eFuse, it actually just tells the FPGA _if it should_ turn on/off the eFuse when it sees a module inserted. So if there is no module, the power will be disabled regardless if the bit is set.